builder/openstack/ssh.go

```diff
@@ -79,7 +81,13 @@ func SSHConfig(username string) func(multistep.StateBag) (*ssh.ClientConfig, err
 	}
 }
 
-func sshAddrFromPool(s *servers.Server, desired string) string {
+func sshAddrFromPool(s *servers.Server, desired string, sshIPVersion string) string {
+
+	ipVersion, err := strconv.Atoi(sshIPVersion)
+	if err != nil {
+		log.Printf("[ERROR] Invalid ssh IP version: %s", sshIPVersion)
+	}
```

Comment:
The validation of the input should go into [prepare](https://github.com/mitchellh/packer/blob/master/builder/openstack/run_config.go#L36) so that you can run ```packer validate template.json``` and verify that your template I syntactically correct.

Setting anything except "", "4", or "6" should result in a verification error.